packages/auth/src/helpers/intl-enzyme-test-helper.js

```diff
@@ -0,0 +1,23 @@
+import { IntlProvider } from 'react-intl';
```

Comment:
Good job!!!
let's change the file name to `intlEnzyme`, we will be using camel case instead of dashes